skill-tree/src/test/java/com/RDS/skilltree/EndorsementsIntegrationTests.java

```diff
@@ -290,4 +290,96 @@ public void testAPIReturn404_OnEndorsementNotFound() {
                 .body("message", equalTo("Endorsement not found"));
     }
 
+    @Test
+    @DisplayName("Return 200 with the endorsements of a particular user given userID")
+    public void testAPIReturn200_OnEndorsementSearchByUserID() {
```

Comment:
Please change function name, give more meaningful

Reply:
I've changed the function names, can you please check once?